faiss/gpu/test/CMakeLists.txt

```diff
@@ -17,12 +17,16 @@
 # the License.
 # =============================================================================
 
-find_package(CUDAToolkit REQUIRED)
-
 # Defines `gtest_discover_tests()`.
 include(GoogleTest)
 add_library(faiss_gpu_test_helper TestUtils.cpp)
+if(USE_ROCM)
+find_package(HIP REQUIRED)
+target_link_libraries(faiss_gpu_test_helper PUBLIC faiss gtest hip::host)
+else()
+find_package(CUDAToolkit REQUIRED)
```

Comment:
Same as above:
1. Can we rely on top-level find package call?
2. Can we use conditional inclusion like we do with RAFT?